src/base/detoken.lua

```diff
@@ -49,6 +49,7 @@
 			-- convert the token into a function to execute
 			local func, err = loadstring("return " .. token)
 			if not func then
+				print("load error:", err)
```

Comment:
Personally, I prefer having one error message, but since it's only two messages it's probably not a big deal :)